tools/update-gateway-settings.ts

```diff
@@ -75,8 +79,8 @@ import { arweave, getContractManifest, initialize, warp } from './utilities';
   const writeInteraction = await contract.writeInteraction(
     {
       function: 'updateGatewaySettings',
-      label,
-      fqdn,
+      // label,
+      // fqdn,
```

Comment:
intended to be commented?

Reply:
yeah - people keep unintentionally setting these